test/common/globalBefore.js

```diff
@@ -61,7 +61,7 @@ function waitUntilBlockchainReady (cb, retries, timeout, baseUrl) {
 
 	baseUrl = baseUrl || 'http://' + config.address + ':' + config.httpPort;
 	(function fetchBlockchainStatus () {
-		popsicle.get(baseUrl + '/api/loader/status')
+		popsicle.get(baseUrl + '/api/node/status')
```

Comment:
Don't hardcode `/api/` as base path any where in tests. Its a configuration in swagger file and we should use it.

Reply:
I've followed the same existing way. If we have introduced a new one, we should create a PR addressing all the cases.

Reply:
Same existing way is what we are refactoring with the help of swagger. So its better to update these to the code places we touch.

Reply:
Agree with @nazarhussain in this case.